commit 6c14fb17e4be70e0f3c03d2f9f6c540b6d5bdcbe
Author: Peter Eisentraut <peter@eisentraut.org>
Date:   Mon Feb 6 22:58:44 2023 +0100

    Fix more outdated comments
    
    Same as in f5da3d8 but for write_relcache_init_file(), the comments
    had gotten a bit wrong due to code added over time.

diff --git a/src/backend/utils/cache/relcache.c b/src/backend/utils/cache/relcache.c
index b57a20c86e0..13f79873733 100644
--- a/src/backend/utils/cache/relcache.c
+++ b/src/backend/utils/cache/relcache.c
@@ -6528,38 +6528,38 @@ write_relcache_init_file(bool shared)
 			/* we assume this was created by heap_copytuple! */
 			write_item(rel->rd_indextuple,
 					   HEAPTUPLESIZE + rel->rd_indextuple->t_len,
 					   fp);
 
-			/* next, write the vector of opfamily OIDs */
+			/* write the vector of opfamily OIDs */
 			write_item(rel->rd_opfamily,
 					   relform->relnatts * sizeof(Oid),
 					   fp);
 
-			/* next, write the vector of opcintype OIDs */
+			/* write the vector of opcintype OIDs */
 			write_item(rel->rd_opcintype,
 					   relform->relnatts * sizeof(Oid),
 					   fp);
 
-			/* next, write the vector of support procedure OIDs */
+			/* write the vector of support procedure OIDs */
 			write_item(rel->rd_support,
 					   relform->relnatts * (rel->rd_indam->amsupport * sizeof(RegProcedure)),
 					   fp);
 
-			/* next, write the vector of collation OIDs */
+			/* write the vector of collation OIDs */
 			write_item(rel->rd_indcollation,
 					   relform->relnatts * sizeof(Oid),
 					   fp);
 
-			/* finally, write the vector of indoption values */
+			/* write the vector of indoption values */
 			write_item(rel->rd_indoption,
 					   relform->relnatts * sizeof(int16),
 					   fp);
 
 			Assert(rel->rd_opcoptions);
 
-			/* finally, write the vector of opcoptions values */
+			/* write the vector of opcoptions values */
 			for (i = 0; i < relform->relnatts; i++)
 			{
 				bytea	   *opt = rel->rd_opcoptions[i];
 
 				write_item(opt, opt ? VARSIZE(opt) : 0, fp);
